mne_bids/mne_bids.py

```diff
@@ -319,6 +320,8 @@ def _coordsystem_json(raw, unit, orient, manufacturer, fname, verbose):
         Used to define the coordinate system for the MEG sensors.
     fname : str
         Filename to save the coordsystem.json to.
+    extra_data : dict
+        A dictionary containing any extra data required in the various files.
```

Comment:
why don't you still use the `raw.info` as I suggested before?

Reply:
Sorry, I hadn't done any work on this really since then.
I will fix it up now :)